src/stories/containers/users/users-manager/users-manager.tsx

```diff
@@ -144,7 +147,7 @@ export default () => {
                   checked={user.active}
                   handleDeleteAccount={() => handleDeleteAccount(user.id)}
                   role={user.roles[0].name}
-                  user={capitalizeWord(user?.username) || ''}
+                  user={capitalizeWordWithoutConvertLowerCase(user?.username) || ''}
```

Comment:
Please, note that `capitalizeWordWithoutConvertLowerCase` will throw an error if the user is empty/undefined, the utility function should be modified to allow empty values

Reply:
yes, my bad, let me put the empty string  in the correct place